hax/treemakers/trigger.py

```diff
@@ -181,45 +181,76 @@ def extract_data(self, event):
         return result
 
 
+# new minitrees
 class TailCut(hax.minitrees.TreeMaker):
-    __version__ = '0.0.3'
+    __version__ = '0.3'
     never_store = True
-
     def get_data(self, dataset, event_list=None):
-        look_back = 100
+        look_back = 50
         # Load Fundamentals and LargestPeakProperties
         # Using load_single_dataset instead of load will ensure no blindding cut is applies
-        data, _ = hax.minitrees.load_single_dataset(dataset, ['Fundamentals', 'LargestPeakProperties'])
-
+        data, _ = hax.minitrees.load_single_dataset(dataset, ['Fundamentals', 'LoneSignals'])#'LargestPeakProperties'])
         if data.empty:
             return pd.DataFrame({})
-
         # Get largest S2 in the event (or 0, if no S2 was found)
-        s2 = data['s2_area'].values
+        s2 = data['s2_0_area'].values
+        #s2 = data['total_peak_area'].values
+        s2_x = data['s2_0_x_nn_tf'].values
+        s2_y = data['s2_0_y_nn_tf'].values
+
+        s2_other = data['s2_1_area'].values
+        s2_x_other = data['s2_1_x_nn_tf'].values
+        s2_y_other = data['s2_1_y_nn_tf'].values
+        
         s2[np.isnan(s2)] = 0
-
+        s2_other[np.isnan(s2_other)] = 0
         # Get the center time of the event
-        t = data['event_time'].values + data['event_duration'].values/2
-
+        t = data['event_time'].values # + data['event_duration'].values/2  #why doesn't it work????
+        tdiff_min = np.zeros(len(t))
+        tdiff_min[0] = 1e18
+        for i in range(1, len(t)):
+            tdiff_min[i] = t[i] - t[i-1]
         # Compute S2/tdif for each value of lookback (from 1 to look_back, inclusive)
         # Allocates n_events * look_back floats; should not be a problem unless look_back is insane number.
         s2_over_tdiff_lookback = np.zeros((len(t), look_back + 1))
         s2_area_lookback = np.zeros((len(t), look_back + 1))
+        se_probabal_coor = np.zeros((len(t), 2, look_back + 1))
```

Comment:
This will make the first 50 (look back) events have at least a 0 se_probabal_coor, and thus those events would be cut away. Maybe detail in docstring.

Reply:
It would be good if you can make a fix. But indeed I'm cutting the 1st 100 events in every datasets because we can not calculating their Tail property.